dashboard/templates/base.html

```diff
@@ -0,0 +1,42 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% block title %} {% endblock %}</title>
+    <!-- Common CSS -->
+    <link rel="stylesheet" href="{% static 'bootstrap/dist/css/bootstrap.css' %}">
+    {% block css %} {% endblock %}
+</head>
+<body class="container-fluid">
+    <header>Student Dashboard :: 
+        {% if user.is_authenticated %}Welcome, {{ user.get_username }}. You can <a href="/accounts/logout">logout here</a>.
+        {% else %}Welcome! Please <a href="/accounts/login">login</a>
+        {% endif %}
+        <hr>
+    </header>
+    {% block content %}
+    {% endblock %}
+    <footer><hr>Copyright 2018 University of Michigan ITS Teaching and Learning</footer>
+</body>
+<!-- Common Javascript -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-124181744-1"></script>
```

Comment:
I tried to get the GA track id out to configuration, but was not successful. 

Putting it into the single file make a progress forward, at least.

Reply:
Yeah! It should be picked up by all the pages anyway. I think I can move this out now, or maybe after I fix #71 . That is up next!